build.sh

```diff
@@ -83,4 +73,4 @@ fi
 cp -R third_party/docker/mkimage* mkdebootstrap/
 
 envsubst < mkdebootstrap/Dockerfile.in > mkdebootstrap/Dockerfile
-gcloud container builds submit . --config="$CONFIG" --verbosity=info --gcs-source-staging-dir="gs://$BUCKET/staging" --gcs-log-dir="gs://$BUCKET/logs" --substitutions=_REPO="$REPO",_TAG="$TAG",_VERSION="$VERSION",_VERSION_NUMBER="$VERSION_NUMBER"
+gcloud container builds submit . --config="$CONFIG" --verbosity=info --substitutions=_REPO="$REPO",_TAG="$TAG",_VERSION="$VERSION",_VERSION_NUMBER="$VERSION_NUMBER"
```

Comment:
We don't currently have have any code that assumes the location of the staging directory, it was strictly for container builder to make the right choice. Now that it does, we don't need to be explicitly passing this in.